source/basic.tex

```diff
@@ -6202,15 +6202,15 @@
 The ``inter-thread happens before'' relation describes arbitrary
 concatenations of ``sequenced before'', ``synchronizes with'' and
 ``dependency-ordered before'' relationships, with two exceptions. The first
-exception is that a concatenation is not permitted to end with
+exception is that a concatenation does not end with
```

Comment:
I feel like "never ends with" would express the original intent a bit better, without sounding like it's giving or removing permission to do something.

Reply:
That's great, thanks!